index.html

```diff
@@ -1182,9 +1182,14 @@ <h2>
         </li>
 
         <li>
-Each public key MUST include <code>id</code> and <code>type</code>
-properties, and exactly one value property. The array of public keys
-MUST NOT contain duplicate entries with the same <code>id</code>.
+Each public key MAY include an <code>id</code> property. The array
+of public keys MUST NOT contain duplicate entries with the same
```

Comment:
This means that DID Document processors must throw an error when they hit multiple entries with the same `id`?

Reply:
I'd say yes.. Even though it wouldn't really be a JSON-LD error, it should still be thrown as a DID Document error.

Reply:
Agreed.